test/test_watch_schedule_builder/test_watch_schedule_builder.py

```diff
@@ -0,0 +1,100 @@
+from ketchup import watch_schedule_builder as wsb
+import datetime as dt
+
+
+def test_calculate_wsb_valid_response():
+    end_date = dt.datetime(2024, 11, 11)
+    start_date = dt.datetime(2024, 11, 1)
+    response = wsb.calculate_watch_schedule(50, end_date, start_date)
+
+    assert response["base_eps_per_day"] == 5
+    assert response["days"] == 10
+    assert response["is_possible"] is True
+    # TODO: Finish my assertions, cool.
```

Comment:
Unsure on why this still exists within the commit, I already removed it from my local file. Somehow it continued to be tracked.

Reply:
we talked about adding all of the response items in assertions here, so there are only 3 right now. you'll have to add those 3 and push another commit up.

Reply:
Ah sorry, I thought you meant per test. Yeah, that's fine. Will get that sorted in the next pull-request.